src/Polly.Core/Simmy/Latency/LatencyChaosPipelineBuilderExtensions.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.Diagnostics.CodeAnalysis;
+using Polly.Simmy.Latency;
+
+namespace Polly.Simmy;
+
+/// <summary>
+/// Extension methods for adding latency to a <see cref="ResiliencePipelineBuilderBase"/>.
+/// </summary>
+public static class LatencyChaosPipelineBuilderExtensions
+{
+    /// <summary>
+    /// Adds a latency chaos strategy to the builder.
+    /// </summary>
+    /// <typeparam name="TBuilder">The builder type.</typeparam>
+    /// <param name="builder">The builder instance.</param>
+    /// <param name="enabled">A value that indicates whether or not the chaos strategy is enabled for a given execution.</param>
+    /// <param name="injectionRate">The injection rate for a given execution, which the value should be between [0, 1].</param>
```

Comment:
Same here about inclusive/exclusive - I think it's implied by the `[0, 1]` notation, but not all users will necessarily be familiar with it.